projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/KubernetesService.java

```diff
@@ -820,42 +820,6 @@ public Set<String> listCronJobs() throws ApiException {
         .collect(Collectors.toSet());
   }
 
-  public void createCronJob(
-      String name,
-      String image,
-      String schedule,
-      boolean enable,
-      V1Container jobContainer,
-      V1Container initContainer,
-      List<V1Volume> volumes)
-      throws ApiException {
-    if (getK8sSupportsV1CronJob()) {
-      createV1CronJob(
-          name,
-          image,
-          schedule,
-          enable,
-          jobContainer,
-          initContainer,
-          volumes,
-          Collections.emptyMap(),
-          Collections.emptyMap(),
-          List.of(""));
-    } else {
-      createV1beta1CronJob(
-          name,
-          image,
-          schedule,
-          enable,
-          jobContainer,
-          initContainer,
-          volumes,
-          Collections.emptyMap(),
-          Collections.emptyMap(),
-          List.of(""));
-    }
-  }
-
   public void createCronJob(
       String name,
       String image,
```

Comment:
nit: so you are deleting those overloaded methods, and tracing to rewrite aall the old 7parameters-long usages to the 10params-long usages?

Isn't it shorter, to adapt the 7parameters-long method to the 10par one directly (fix the impl to shortcut and reuse the broader one)? No need to rewrite each and every usage...